app/Http/Controllers/Measurements/MeasurementController.php

```diff
@@ -24,11 +29,12 @@ public function __construct(MeasurementRepositoryInterface $measurementRepositor
     /**
      * List all measurements
      *
-     * @param int $measurementId
+     * @param int $deviceUuid
```

Comment:
Voor consistentie zal ik die zo aanpassen, maar functioneel doet het er niet toe; de waarde is er nog steeds